src/main/java/com/alrex/parcool/common/network/SyncServerInfoMessage.java

```diff
@@ -21,7 +21,6 @@
 import java.util.function.Supplier;
 
 public class SyncServerInfoMessage {
-	private static final Logger log = LoggerFactory.getLogger(SyncServerInfoMessage.class);
```

Comment:
Not part of the change, but I was getting an error in this line, I don't know why.  Its removal is in the second commit, I can undo it if requested

Reply:
All ParCool classes uses `com.alrex.parcool.ParCool.LOGGER` across the project so you don't need to create other logging instance.

Reply:
I don't remember adding this line anyway its ok to remove it